runtime/development/src/lib.rs

```diff
@@ -1764,6 +1773,12 @@ construct_runtime!(
 		Migration: pallet_migration_manager::{Pallet, Call, Storage, Event<T>} = 199,
 		// admin stuff
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 200,
+
+		// EVM pallets
+		EVM: pallet_evm::{Pallet, Config, Call, Storage, Event<T>} = 160,
```

Comment:
It seems that we are indexing the evm pallets with a lower number than pallets above that; is that intentional? Shouldn't we move them up to be sorted ASC?

Reply:
I think that's fine but I would still move it up to match the order

Reply:
Yes, we should move it up in this case. However, we should always be careful with these things as the order of the pallet in the `construct_runtime` macro decide the order of the `on_initialize` (in-order) and `on_finalize` (reverse of order) executions. 

So for the crucial part, the indices are irrelevant. Moreover, indices should not be swapped and most ideally not be reused. The later part will be interesting to see in the future, given we only have `u8` supported. But Kusama/Polkadot will hit this limitation much earlier 😅

Reply:
I've been trying a bit to think about how the separation of "API stability" and "ABI stability" that I'm familiar with from the C world map to Substrate - things like indices I think are a big part of this. Let's chat about it some this week if we have time :)